lib/git_reflow/git_server/pull_request.rb

```diff
@@ -140,6 +140,25 @@ def method_missing(method_sym, *arguments, &block)
           super
         end
       end
+
+      def merge_feature_branch(feature_branch_name, options = {})
```

Comment:
also, since you're removing the local syncs, you'll have to add them back to this method to ensure BitBucket continues to update the feature branch's dependencies.  These were 4 lines I found removed that we'll need to continue to support here, but just double check me :wink:

```ruby
update_current_branch
fetch_destination(base_branch)
update_destination(feature_branch)
update_destination(base_branch)
```